hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/distributions/dstable/DistributionSetTable.java

```diff
@@ -186,17 +184,23 @@ protected Object getItemIdToSelect() {
     }
 
     @Override
-    protected ManageDistUIState getManagmentEntityState() {
+    protected ManageDistUIState getManagementEntityState() {
         return manageDistUIState;
     }
 
     @Override
-    protected void publishEntityAfterValueChange(final DistributionSet distributionSet) {
-        eventBus.publish(this, new DistributionTableEvent(BaseEntityEventType.SELECTED_ENTITY, distributionSet));
+    protected void publishAdditionalEvent() {
```

Comment:
Is this really required?  `publishSelectedEntityEvent()`will be called anyway by `AbstractTable#selectEntity(...)` so I think you can merge them like before, right?